packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/project/controllers/ProjectRestController.java

```diff
@@ -47,6 +51,8 @@
 @RequestMapping("/api/rest/projects")
 public class ProjectRestController {
 
+    private static final OffsetDateTime DEFAULT_CREATED = Instant.EPOCH.atOffset(ZoneOffset.UTC);
```

Comment:
In this very specific case, you could simply add `createdOn` to `ProjectDTO` and (the GraphQL schema) and retrieve this information. We will display it on the frontend one day or another.... but it can be done later.

Reply:
Ok I will do that later, thank you